pkg/promlib/README.md

```diff
@@ -0,0 +1,11 @@
+# promlib
+
+Prometheus Library a.k.a. (promlib) is the foundation of the Grafana prometheus data source backend.
+
+### How to tag/version?
+- Checkout the commit you want to tag (git checkout <COMMIT_SHA>)
```

Comment:
nit: per `markdownlint` [md001](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md001.md) and [md022](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md022.md) / [md032](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md032.md),

```suggestion
## How to tag/version?

- Checkout the commit you want to tag (git checkout <COMMIT_SHA>)
```